python/lsst/ts/xml/data/sal_interfaces/TunableLaser/TunableLaser_Events.xml

```diff
@@ -34,6 +34,18 @@ LaserErrorCode_HW_CPU_ERROR = 7304
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_logevent_alignmentChanged</EFDB_Topic>
+    <item>
+      <EFDB_Name>optical_configuration</EFDB_Name>
```

Comment:
Same as above, topic attributes should be `lowerCamelCase`.. Also, consider renaming this just `configuration` this way, you have a command `setOpticalConfiguration` with a `configuration` attribute and an `opticalConfiguration` event with `configuration` attribute. It would all match up.